firmware/boron-ins-fsm/test/consoleFunctionTests.cpp

```diff
@@ -4,6 +4,9 @@
 #include "../src/consoleFunctions.h"
 #include "../src/flashAddresses.h"
 
+// Define globalDoorID only for console function tests
+IMDoorID globalDoorID;
```

Comment:
This was originally defined in base.h, but since this variable was already defined in imDoorSensor.cpp, it was giving duplicate definition errors while trying to compile the door sensor tests